### PR TITLE
Made util into a class.

### PR DESCRIPTION
**Pull request by ItsJordan**
May or may not have just copy pasted the code from my desktop because I'm bad at git command line ._.

### CODE REVIEW

File: core/Util.js [hunk 1]
```diff
@@ -4,82 +4,86 @@ const client = require("../index");
 const config = require("../config");
 const { RichEmbed } = require("discord.js");
 
+module.exports = class Util {
 // Logging Time Format
```
Comment (kurisubrooks) [inline core/Util.js]: Indent

File: core/Util.js [hunk 2]
```diff
@@ -4,82 +4,86 @@ const client = require("../index");
...
+    // Handle User Join
+    static handleJoin(member) {
+        if (config.adminServer.includes(member.guild.id)) {
+            member.addRole(member.guild.roles.find("name", "Muggle"));
+            member.send("", { embed: { "description": "Welcome to Kurisu's Server!\nTo get started, I kindly ask you take the following quizzes,\nand post the results in #general, so you can be sorted in to your appropriate roles!\n \nhttps://my.pottermore.com/user-profile/my-house/ilvermorny \nhttps://my.pottermore.com/user-profile/my-house/hogwarts\n\nWe don't have a set series of rules as we're a relatively small server,\nbut I do kindly ask that you don't spam, be mature and don't troll.\n\nSincerely, Kurisu." } });
+        }
     }
 };
 
 // Unhandled Promise Rejections
```
Comment (kurisubrooks) [inline core/Util.js]: You can't call `this` from outside a class, I don't think?

File: core/Util.js (hunk 1 above)
Comment (kurisubrooks) [inline core/Util.js]: just indent this to 4 spaces

File: core/Util.js (hunk 2 above)
Comment (ItsJordan) [inline core/Util.js]: According to grey you can?

Comment (kurisubrooks) [inline core/Util.js]: No.

**Pull request merged**